content/post/074-explore-the-files-api-on-protoschool.md

```diff
@@ -0,0 +1,42 @@
+
+If you're new to IPFS and finding CIDs to be somewhat unfriendly as a naming convention, keep an eye out in this tutorial for an explanation of how you can use the `wrapWithDirectory` option in the Regular Files API to create more human-readable paths and filenames.
+
+### More to explore on ProtoSchool
+
+#### Explore other IPFS APIs through coding challenges
+Like most of our tutorials, this latest addition uses JavaScript coding challenges to teach IPFS methods, allowing you to explore them right from your web browser without installing anything or touching the command line. Check out our other code-based IPFS tutorials on ProtoSchool:
+
+- The [Mutable File System (MFS)](https://proto.school/#/mutable-file-system) (part of the Files API)
+- The DAG API: [P2P Data Links with Content Addressing](https://proto.school/#/basics)
+- The DAG API: [Blogging on the Decentralized Web](https://proto.school/#/blog)
+
+#### Don't know JavaScript or don't know how to code?
+Our [Decentralized Data Structures](https://proto.school/#/data-structures) tutorial offers a code-free look at important concepts like content addressing and hashing.
+
+#### Prefer to learn in a group setting?
+Find a [ProtoSchool chapter near you](https://proto.school/#/chapters) and attend a live workshop.
+
+#### Help us improve our curriculum
+We're constantly working to improve and grow our ProtoSchool curriculum, and have lots of new content planned for you in the months ahead. You can help us improve our offereings by sharing your feedback on existing tutorials (using the links provided a the bottom of each lesson) or by [building a new tutorial](https://proto.school/#/build). (Our platform currently supports browser-based JavaScript coding challenges, text-based lessons, and multiple-choice quizzes.)
+
+### Other resources on using IPFS with files
+
+If you'd like to dig a little deeper, we recommend checking out the **Understanding How IPFS Deals with Files** course offered at IPFS Camp 2019. (No coding required.) It offers a deep dive on key concepts like immutability, content addressing, hashing, the anatomy of CIDs, what the heck a Merkle DAG is, and how chunk size affects file imports. It also covers the joys and pitfalls of the Mutable File System (MFS), the layer of abstraction in IPFS that lets you work with immutable files and directories as if you were using a traditional name-based file system. As with all great talks, this one is enhanced by images of adorable poodles. 🐩 (12 / 10, would recommend.)
```

Comment:
maybe add that it also demonstrates how to use cid.ipfs.io and dag.ipfs.io to introspect into what IPFS is doing behind the scenes to manipulate files?

Reply:
Added!